core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptMapCompareRequestBuilder.java

```diff
@@ -30,20 +33,23 @@
 	
```

Comment:
`5000` limit is present in this class.

Reply:
This only limits the size of the result, the other one is for checking whether a set is larger than the default 50 000 limit.